zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,25 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Honeybadger will use the `honeybadger` stream by default if no stream is given
+in the URL. Make sure the selected stream exists in Zulip.
+
+**Follow these steps:**
+
+1. Go to `Alerts & Integrations` page of your `Honeybadger` Console.
+
+2. Find and select the `WebHook` integration option.
+![](/static/images/integrations/honeybadger/002.png)
+
+3. Enter the URL constructed above into the URL field and configure
+the triggers for the webhook.
+![](/static/images/integrations/honeybadger/003.png)
+
+4. Test the webhook by clicking `Test This Integration`.
```

Comment:
How about "Test the webhook by clicking on the `Test This Integration` button at the bottom of the page".

Makes it a bit easier to find I think

Reply:
will do